app/javascript/controllers/selection_controller.js

```diff
@@ -9,10 +9,20 @@ export default class extends Controller {
   static outlets = ["action-link"];
 
   connect() {
+    // When you are first routed to a sample#show page, the path will not contain any query information about the
```

Comment:
**suggestion:** I think you might have other thought this a little too much.  What I think would be better is to revert this file back to the original then just set the `data-storage-key` attribute (which actually should be re-written as a stimulus `value` I think).  That will pass in what ever key you want to use for storage without having to hack this.  This selection should be good for any table that has the capability to  select items.

Reply:
Thanks for this, much simpler. Changed in 47a3ba31d176c4f5e799487e1606f6263374f109